### ISSUE TITLE
"Back to cookbook" navigates to authorization page

### ISSUE THREAD

**Issue by angelo-v** [NoelDeMartin/umai#14]
1. I visit Umai and sign in
2. Go to a recipe page
3. Refresh the page (F5)
4. -> I land on the authorization page of my CSS instance and authorize again
5. I am back on the recipe page
6. I click "Back to cookbook"
7. Instead of going back to the cookbook I am back on the CSS authorization page

**Comment by NoelDeMartin** [comment]
Thanks for reporting, I've also noticed this myself but I hadn't looked into it.

It seems like the back functionality is doing a browser back rather than checking if the previous page was the cookbook. Initially I followed this approach because I didn't want to add new navigation entries to the history for going back, but maybe it'd be better to just do a normal "forward" navigation back to the cookbook.
